k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -747,6 +749,18 @@ private void onDiscard() {
         }
     }
 
+    private void onUrgent(MenuItem item) {
```

Comment:
@philipwhiuk , Does this look okay? 
and I have changed all the isHighPriority to isUrgent.